recipes/datamol/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "datamol" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datamol-{{ version }}.tar.gz
+  sha256: 600d2462b535cfe176b321354d6dc335239a0012b6f43b98ee64d65bffd847f5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - fsspec >=0.8.0
+    - joblib
+    - loguru
+    - numpy
+    - pandas
+    - pillow
+    - python >=3.7
+    - scipy
+    - selfies
+    - tqdm
+    - rdkit >=2020.09
+
+test:
+  imports:
+    - datamol
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/datamol-org/datamol
+  summary: A python library to work with molecules. Built on top of RDKit.
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - hadim
+    - maclandrol
```

Comment:
@maclandrol are you OK being a maintainer here?

Reply:
He reacted above at https://github.com/conda-forge/staged-recipes/pull/14485#issuecomment-814880595

Reply:
@ocefpaf  yes. I confirm